modin/core/execution/dask/implementations/pandas_on_dask/io/io.py

```diff
@@ -13,6 +13,9 @@
 
 """Module houses class that implements ``BaseIO`` using Dask as an execution engine."""
 
+import dask.dataframe as dd
+from distributed.client import default_client
```

Comment:
Let's use `DaskWrapper` to interact with Dask API.

Reply:
Are you sure? As I see it, we use the public API without wrapping it in Wrapper, for both dask and ray.
Also `DaskWrapper` has annotation: "The class responsible for execution of remote operations.", but creating a Dask Dataframe is not valid for it.

Considering your wishes, I can suggest changing `import dask.dataframe as dd` to `from dask.dataframe import from_delayed`. What do you think about it?

Reply:
> Are you sure? As I see it, we use the public API without wrapping it in Wrapper, for both dask and ray.
Also DaskWrapper has annotation: "The class responsible for execution of remote operations.", but creating a Dask Dataframe is not valid for it.

Yes, I'm sure. It's okay to change the description because that's what these classes were designed for. All places where the API is used directly need to be updated to work through a wrapper.

Reply:
Ok, I see reasons for this, so I just want to clarify. In this case, we just create the same method in `DaskWrapper` that proxies it to DaskDatafame, like this:
```python
@classmethod
def from_delayed(cls, partitions):
    dd.from_delayed(partitions)
```
Is correct?

Do I need to create an issue to refactor another public API calls?

Reply:
> Do I need to create an issue to refactor another public API calls?

It would be great.

> In this case, we just create the same method in DaskWrapper that proxies it to DaskDatafame, like this:

We discussed it offline, this can also be done separately.